cross_compile/ros2_cross_compile.py

```diff
@@ -46,18 +46,18 @@ def create_arg_parser():
         choices=['ubuntu', 'debian'],
         help='Target OS')
     parser.add_argument(
-        '-d', '--distro',
+        '-d', '--rosdistro',
         required=False,
         type=str,
         default='dashing',
-        choices=['ardent', 'bouncy', 'crystal', 'dashing'],
+        choices=['dashing', 'eloquent', 'kinetic', 'melodic'],
```

Comment:
To reduce duplication, perhaps `Platform._SUPPORTED_ROS_DISTROS + Platform._SUPPORTED_ROS2_DISTROS` for the choices here